packages/stockflux-watchlist/public/notificationHandler.js

```diff
@@ -1,4 +1,4 @@
-// eslint-disable-next-line no-unused-vars
+/*eslint-disable-next-line no-unused-vars*/
```

Comment:
I don't see any unused vars here. Might be worth just removing this comment. I wonder why eslint insisted on changing this to a multiline style comment `/**/` instead of single line style `//`

Reply:
It's needed for the function itself as eslint thinks it's unused
![image](https://user-images.githubusercontent.com/32611900/65140163-e00ab300-da05-11e9-9b90-2f6e0f2abda7.png)

Also changed it to single line comment now.